.gitignore

```diff
@@ -71,3 +71,12 @@ c/
 x64-effects/
 Dockerfile
 docker.nix
+
+# Ignore tool repos
+tools/repos/
```

Comment:
Hmm, it's not exactly a typical `vendor` folder though... These aren't tools that are shipped _with_ the harness, but repositories that the harness manages for you. In particular, this could contain a separate `git worktree` of your main `wasmtime` repo elsewhere.

Reply:
OK, how about `ext` or `external` then?